assignments/lambda-classes.js

```diff
@@ -1 +1,197 @@
 // CODE here for your Lambda Classes
+
+
+// ***********************************************************
+// ******************       CLASSES       ***********************
+// ***********************************************************
+class Person {
+  constructor(props){
+    this.name = props.name;
+    this.age = props.age;
+    this.location = props.location;
+    this.gender = props.gender;
+  };
+  speak() {
+    return `Hello my name is ${this.name}, I am from ${this.location}`
+  };
+}
+
+class Instructor extends Person {
+  constructor(props){
+    super(props);
+    this.specialty = props.specialty;
+    this.favLanguage = props.favLanguage;
+    this.catchPhrase = props.catchPhrase;
+  }
+  demo(subject) {
+    return `Today we are learning about ${subject}`;    
+  };
+  grade(student, subject) {
+    return `${student.name} receives a perfect score on ${subject}`
+  };
+  gradeProjects(student){
+    let rand = Math.round(Math.random() * 100);
```

Comment:
I like that you declared the random number as it's own variable, makes it easier to edit it down the line.